mappings/net/minecraft/entity/mob/ZombieEntity.mapping

```diff
@@ -1,25 +1,26 @@
 CLASS aus net/minecraft/entity/mob/ZombieEntity
 	CLASS aus$a DestroyEggGoal
-	FIELD bB TICKS_UNTIL_DROWNED Lqg;
+	FIELD bB CONVERTING_IN_WATER Lqg;
```

Comment:
This is an int, not boolean, so it should be `TICKS_UNTIL_CONVERSION_IN_WATER`.

Reply:
Is it a count up or a count down?

Reply:
No, this is a boolean in a data tracker parameter. The remapper failed to recognize this field change.

Reply:
Though is the `_IN_WATER` suffix necessary?

Reply:
Yes. The setter call is in a `isIn(FluidTags.WATER)` check.